client/src/components/Dashboard/LiveTicker.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import axios from 'axios'
+import TickerStar from './TickerStar'
+import { Loading, Row, TickerContainer, StockSymbol, Star } from '../Styles/Dashboard/LiveTickerStyles' 
+
+class LiveTicker extends React.Component{
+    constructor(){
+        super();
+        this.state = {
+            timeStamp: {},
+            companies: ['DJI', 'NDAQ', 'SPX', 'AAPL', 'AMZN'], // stock company symbols
```

Comment:
At some point we should either integrate search or have an array of all of the S&P stocks, etc. (I know that would probably take a long time to load) [does that exceed the API limit?]

Reply:
I also agree with this point. For now, we will have to add the search bar. Unfortunately, the full list will make things buggy with alpha vantage due to the limited calls. If time permits, I think the best thing to do would be implementing the other API that you suggested. I noticed that the methods for calling that particular API were quite a bit different and would require some refactoring on my part. If we arrive at a point where it seems realistic that the switch can be made, I think that would be the way to go. We'd definitely be able to load entire lists that way and elevate the quality.

Reply:
Is there a way where we can randomly display the stocks every 5 minutes? But only 5 at a time?